fs/cromfs/cromfs.h

```diff
@@ -116,6 +116,6 @@ struct cromfs_node_s
     uint32_t cn_link;    /* Offset to an arbitrary node (for hard link) */
     uint32_t cn_blocks;  /* Offset to first block of compressed data (for read) */
   } u;
-};
+} __attribute__((packed)); /* Added to resolve hardFault caused by unaligned memory access of cromfs nodes.
```

Comment:
@vxj9800 - there is a compiler.h macros the do this in a portable way see https://github.com/apache/incubator-nuttx/commit/440ca487789fcba776fc04e90f0ce71b4c1e86d3

Reply:
Done.